config/x_hh_bbtautau_run2.yaml

```diff
@@ -24,16 +28,1318 @@ processes:
+    - TTTJ
+    - TTTT
+    - TTTW
+    - TT4b
+  - process: W
+  - process: EWK
+    allow_zero_integral: True
+    allow_negative_bins_within_error: True
+    subprocesses:
+    - EWK_WplusToLNu
+    - EWK_WminusToLNu
+    - EWK_ZTo2L
+    - EWK_ZTo2Nu
+  - process: VVV
+    allow_zero_integral: True
+    max_n_sigma_for_negative_bins: 1.01
+    allow_negative_bins_within_error: True
+    subprocesses:
+    - WWW_4F
+    - WWZ_4F
+    - WZZ
+    - ZZZ
+  - process: QCD
+    allow_zero_integral: True
+    allow_negative_integral: True
+    allow_negative_bins_within_error: True
+    max_n_sigma_for_negative_bins: 10
+  - process: ggH
+    allow_zero_integral: True
+    allow_negative_bins_within_error: True
+    subprocesses:
+    - GluGluHToTauTau_M125
+    - ggHToZZTo2L2Q
+    - GluGluHToWWTo2L2Nu_M125
+  - process: qqH
+    allow_zero_integral: True
+    allow_negative_bins_within_error: True
+    subprocesses:
+    - VBFHToTauTau_M125
+    - VBFHToWWTo2L2Nu_M125
+    - VBFHToWWTo2L2Nu_M125
+  - process: VH
+    allow_zero_integral: True
+    allow_negative_bins_within_error: True
+    subprocesses:
+    - ZH_Hbb_Zll
+    - ZH_Hbb_Zqq
+    - WplusHToTauTau_M125
+    - WminusHToTauTau_M125
+    - ZHToTauTau_M125
+  - process: ttH
+    allow_zero_integral: True
+    allow_negative_bins_within_error: True
+    subprocesses:
+    - ttHTobb_M125
+    - ttHToTauTau_M125
+
 uncertainties:
   - name: lumi_13TeV
     type: lnN
     value: 0.03
-    processes: ^.*$
-  - name: JER
+  - name: BR_htt
+    type: lnN
+    value: 0.0165
+    processes:
+    - ttH
+    - ggH_htt
+    - qqH_htt
+    - VH_htt
+    - GluGluToRadionToHHTo2B2Tau*
+    - GluGluToBulkGravitonToHHTo2B2Tau*
+  - name: BR_hbb
+    type: lnN
+    value: 0.0124
+    processes:
+    - ZH_Hbb_Zll
+    - ZH_Hbb_Zqq
+    - GluGluToRadionToHHTo2B2Tau*
+    - GluGluToBulkGravitonToHHTo2B2Tau*
+  - name: pdf_gg
+    type: lnN
+    value: 0.042
+  - name: th_ggH
+    type: lnN
+    processes:
+    - ggH
+    value: 0.039
+  - name: th_qqH
+    type: lnN
+    processes:
+    - qqH
+    value: 0.004
+  - name: pdf_Higgs_ggH
+    type: lnN
+    value: 0.019
+    processes:
+    - ggH
+    - GluGluToRadionToHHTo2B2Tau_M_${MX}
+    - GluGluToBulkGravitonToHHTo2B2Tau_M_${MX}
+  - name: pdf_Higgs_ttH
+    type: lnN
+    value: 0.03
+    processes:
+    - ttH
+  - name: pdf_Higgs_tH
+    type: lnN
+    value: 0.047
+    processes:
+    - tH
+  - name: pdf_Higgs_qqH
+    type: lnN
+    value: 0.021 # 0.1
+    processes:
+    - qqH
+
+
+  - name: pdf_Higgs_qqbar
+    type: lnN
+    hasMultipleValues: True
+    value:
+    - value: 0.013
+      type: lnN
+      processes:
+      - ZH_Hbb_Zll
+      - ZH_Hbb_Zqq
+    - value: 0.021
+      type: lnN
+      processes:
+      - VBFHToTauTau_M125
+    - value: 0.017
+      type: lnN
+      processes:
+      - WminusHToTauTau_M125
+      - WplusHToTauTau_M125
+
+  - name: pdf_DY
+    type: lnN
+    value: 0.007296099203254119
+    processes:
+    - qqH
```

Comment:
why pdf_DY is applied to qqH?

Reply:
eh because I did a stupid copy-paste. Fixed